front/components/poke/subscriptions/table.tsx

```diff
@@ -407,15 +407,15 @@ function UpgradeDowngradeModal({
             owner={owner}
           />
         </div>
-        {isProPlanCode(subscription.plan.code) && (
+        {subscription.plan.code.startsWith("PRO_") && (
```

Comment:
Won't apply because I love the fact that in a line you can see it's scrappy as opposite to having the scrappiness hidden in a function. Or I could call it `pokéOnlyIsProSubscription` but in this case we want to do for all (free and enterprise too) and it deserves its own PR!